ecosystem/sep-0031.md

```diff
@@ -103,7 +92,16 @@ sequenceDiagram
 
     Sending Anchor->>+Receiving Anchor: POST [SEP-31]/transactions
     Receiving Anchor-->>Receiving Anchor: checks customer statuses, <br> links quote, <br> creates transaction record
-    Receiving Anchor-->>-Sending Anchor: transaction id, receiving account & memo
+    Receiving Anchor-->>-Sending Anchor: transaction id
+
+    note right of Sending Anchor: If Sending Client or Receiving Client <br >has already been <br> accepted, this section <br> is unnecessary.
```

Comment:
Should we be looping once for the Sending Client and once for the Receiving Client? If we don't then it looks like this says we can move forward with the transfer when the Sending Client has provided KYC even though we don't know anything about the Receiving Client.